lambdas/service/app.py

```diff
@@ -810,69 +885,79 @@ def start_manifest_generation():
                     status_code=301 if wait_time else 302)
 
 
-@app.route('/fetch/manifest/files', methods=['GET'], cors=True)
+@app.route('/fetch/manifest/files', methods=['GET'], cors=True, spec={
+    'description': 'Initiate and check status of a manifest generation job, returning a 200 response with simulated '
+                   'headers in the body.<p>'
```

Comment:
Consider putting the `'<p>'` on their own lines to make the separation even more clear.